jbpm-workitems/jbpm-workitems-email-typed/pom.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0"?>
+<project
```

Comment:
I don't think adding a copy of the existing email workitem here is good. it will confuse users more than anything to have two of the exact same thing. (same comment already added :) )